spec/services/data_migrations/backfill_setup_interviews_permission_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe DataMigrations::BackfillSetupInterviewsPermission do
+  let!(:provider_permissions) { create(:provider_permissions, make_decisions: true, setup_interviews: false) }
+
+  it 'sets the setup_interviews permission to true' do
+    described_class.new.change
+    expect(provider_permissions.reload.setup_interviews).to be(true)
+  end
+
+  it 'does not update the application_choice timestamps' do
+    provider_permissions_created_at = provider_permissions.created_at
+    provider_permissions_updated_at = provider_permissions.updated_at
+
+    described_class.new.change
+
+    expect(provider_permissions.created_at).to eq(provider_permissions_created_at)
+    expect(provider_permissions.updated_at).to eq(provider_permissions_updated_at)
+  end
+
+  it 'does not update the audit log' do
```

Comment:
```suggestion
  it 'does not update the audit log', with_audited: true do
```
this is required to ensure auditing is enabled